docs/_layouts/index.html

```diff
@@ -12,7 +12,7 @@
       <h1 class="project-name">{{ site.title | default: site.github.repository_name }}</h1>
       <h2 class="project-tagline">{{ site.description | default: site.github.project_tagline }}</h2>
       <a href="{{ site.github.repository_url }}" class="btn">View on GitHub</a>
-      <a href="{{ site.github.repository_url }}/releases/tag/0.2.1" class="btn">Download 0.2.1</a>
+      <a href="{{ site.github.repository_url }}/releases/tag/0.3.3" class="btn">Download 0.2.1</a>
```

Comment:
`Download 0.2.1` -> `Download 0.3.3` ?

Reply:
Nice catch. Another reason why we should automate this!